test/weaponEffects.test.js

```diff
@@ -2,16 +2,16 @@ import fs from "fs";
 import path from "path";
```

Comment:
Minor changes to handle new pack structure/dynamic pack generation. No functional changes.